packages/cli-tools/bin/common.ts

```diff
@@ -90,4 +91,20 @@ export function createFnParseInt(name: string): (s: string) => number {
         }
         return n
     }
+}
+
+export const createStreamId = (streamIdOrPath: string|undefined, options: any) => {
+    if (streamIdOrPath === undefined) {
+        return undefined
+    }
+    const PATH_PREFIX = '/'
+    if (!streamIdOrPath.startsWith(PATH_PREFIX)) {
+        return streamIdOrPath
+    }
+    const privateKey = options.privateKey
+    if (privateKey === undefined) {
+        console.error(`relative stream id ${streamIdOrPath} requires authentication`)
+        process.exit(1)
+    }
+    return new Wallet(privateKey).address.toLowerCase() + streamIdOrPath
```

Comment:
Isn't this logic something we wanted to include within streamr-client-javascript itself?

Reply:
Yes, in the future this path support should be in the client. Currently it is not possible to add the support (would require too much refactoring). But we could re-check the situation after Brubeck refactoring has been done.